app/sliders/importer.py

```diff
@@ -63,10 +69,12 @@ def save_to_json(self, parsed_data, outfile_path):
         :param parsed_data: dict, parsed data
         :param outfile_path: str, path to the JSON file where to save the data
         """
+        # Replaces the old outfile if it exists
```

Comment:
Writing a file in "w" mode automatically replaces the old file (if it exists) with the new file, so there is no need to do this extra check

Reply:
You're right. It was an issue with my configuration. The file was created but not saved automatically. Causing it to not visibly update it's content. i did take the lines out again